azure_arc_ml_jumpstart/aks/arm_template/artifacts/azuredeploy.parameters.example.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sshRSAPublicKey": {
+      "value": "ssh-rsa THISISTOPSECRETSTUFF1234XXXXXXXXXXXXXtEbhbVqD bilbo@MacBook-Pro.localdomain"
+    },
+    "spnClientId": {
+      "value": "33333333-XXXX-YYYY-XXXX-YTYTYTYT"
+    },
+    "spnClientSecret": {
+      "value": "33333333-XXXX-YTYT-9c21-7777777777"
+    },
+    "spnTenantId": {
+      "value": "33333333-XXXX-41af-1111-7777777777"
+    },
+    "windowsAdminUsername": {
+      "value": "arcdemo"
+    },
+    "windowsAdminPassword": {
+      "value": "ArcPassword123!!"
+    },
+    "myIpAddress": {
+      "value": "88.88.88.88"
+    },
+    "logAnalyticsWorkspaceName": {
+      "value": "Arc-Data-LA"
+    },
+    "templateBaseUrl": {
+      "value": "https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_ml_jumpstart/aks/arm_template/"
+    },
+    "kubernetesVersion": {
+      "value": "1.19.11"
```

Comment:
Were you able to test a higher version?

Reply:
Tested successfully on K8s 1.21.2:
![image](https://user-images.githubusercontent.com/46581776/130332757-ad2da5ce-db7b-4287-b162-9c312e8a36b6.png)

Reply:
Thanks for verifying. Let's not change this for now as data services are still not supported with 1.2x and above

Reply:
Changing back to `1.19.11`